tutorials/Tutorial13_Question_generation.py

```diff
@@ -31,13 +31,19 @@
 """
 The most basic version of a question generator pipeline takes a document as input and outputs generated questions
 which the the document can answer.
-"""
+# """
 
 # QuestionGenerationPipeline
 question_generation_pipeline = QuestionGenerationPipeline(question_generator)
-for document in document_store:
-        result = question_generation_pipeline.run(documents=[document])
-        pprint(result)
+for idx, document in enumerate(document_store):
+        
+    print(f"\n * Generating questions for document {idx}: {document.content[:50]}...")
+    result = question_generation_pipeline.run(documents=[document])
+
+    print("Generated questions:")
```

Comment:
I like these titles but the printout seems a bit dense. Could you add a line break before "Generated questions"?